packages/rspack/src/builtin-plugin/css-extract/loader.ts

```diff
@@ -79,10 +84,15 @@ export const pitch: LoaderDefinition["pitch"] = function (request, _, data) {
 		this._compiler &&
 		this._compiler.options &&
 		this._compiler.options.experiments &&
-		this._compiler.options.experiments.css
+		this._compiler.options.experiments.css &&
+		this._module &&
+		(this._module.type === "css" ||
+			this._module.type === "css/auto" ||
+			this._module.type === "css/global" ||
+			this._module.type === "css/module")
 	) {
 		let e = new Error(
-			"You can't use `experiments.css` and `css-extract-rspack-plugin` together, please set `experiments.css` to `false`"
+			'You can\'t use `experiments.css` (`experiments.futureDefaults` enable built-in CSS support by default) and `mini-css-extract-plugin` together, please set `experiments.css` to `false` or set `{ type: "javascript/auto" }` for rules with `mini-css-extract-plugin` in your webpack config (now `mini-css-extract-plugin` does nothing).'
```

Comment:
this is weird? why this warning can't use experiments.css with mini-css-extract-plugin together and then tests case use them together?

Reply:
Because one module with type css should not handled by this plugin, only type 'javascript' can